src/views/InvolvementProfile/components/Membership/components/MemberList/index.js

```diff
@@ -33,7 +34,21 @@ const MemberList = ({
     return () => window.removeEventListener('resize', resize);
   });
 
+  // re-arranging names alpabetically and also by Hierarchy "Advisor" > "Leader" >
+  //"member" > "Guest".
+
   const compareByLastThenFirst = (a, b) => {
+    if (a.ParticipationDescription !== b.ParticipationDescription) {
+      if (a.ParticipationDescription == 'Advisor') return -1;
+      if (b.ParticipationDescription == 'Advisor') return 1;
+      if (a.ParticipationDescription == 'Leader') return -1;
+      if (b.ParticipationDescription == 'Leader') return 1;
+      if (a.ParticipationDescription == 'Member') return -1;
+      if (b.ParticipationDescription == 'Member') return 1;
+      if (a.ParticipationDescription == 'Guest') return -1;
+      if (b.ParticipationDescription == 'Guest') return 1;
```

Comment:
If we made it to this point, we've already checked all the cases where `a !== 'Guest'` and `b !== 'Guest'`, so technically I think this code will never be reached.

Reply:
Did I So technically it is unnecessary to add Guest into the list? because all the case has been handled?